.prettierignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything outside of the "src" directory, and also ignore the auto-generated type file
+/*
+!/src
+/src/isaac-data-types.d.ts
```

Comment:
I think you should be alright by just leaving this line and removing the rest: prettier should ignore `node_modules` by default, so we may as well have the whole repo covered by it.

Reply:
I know we had some issues with that on the front end hence why I took this approach, but it seems to work fine here with only the 1 file to ignore!